piecewise_web/js/ndt.js

```diff
@@ -464,11 +367,6 @@ NDTjs.prototype.ndtS2cTest = function (ndtSocket) {
       }
       // Calculation per spec, compared between client and server
       // understanding.
-      for (var web100Variable in messageContent) {
```

Comment:
Is this handled somewhere else now, such that web100 data from the server is recorded properly? Or have is just been decided that we don't actually need this data for any specific purpose at the moment?